make/template/inspircd

```diff
@@ -88,12 +88,11 @@ if (!defined($sub))
 {
 	print STDERR "Invalid command or none given.\n";
 	cmd_help();
-	exit 1;
+	exit 3;	# Unimplemented feature
```

Comment:
@attilamolnar Fair enough.

@StevenVanAcker instead of labelling them you could make them into constants with a link to the spec. Something like this:

```perl 
use constant {
    EXIT_SUCCESS = 0,
    EXIT_PROGRAM_NOT_RUNNING = 3,
}
```